x/ccv/provider/keeper/keeper.go

```diff
@@ -569,8 +572,8 @@ func (k Keeper) ConsumeMaturedUnbondingOps(ctx sdk.Context) []uint64 {
 
 // Retrieves the underlying client state corresponding to a connection ID.
 func (k Keeper) getUnderlyingClient(ctx sdk.Context, connectionID string) (
-	clientID string, tmClient *ibctmtypes.ClientState, err error) {
-
+	clientID string, tmClient *ibctmtypes.ClientState, err error,
```

Comment:
whitespace diff?

Reply:
It's because of gofumpt, I can revert those changes if you prefer.